### ISSUE TITLE
#6536 Screenreader and proposal tiles 

### ISSUE THREAD

**Issue by SandisStudi** [liqd/a4-meinberlin#4659]
**URL:** https://meinberlin-dev.liqd.net/projekte/module/priorisierung-2/?ordering=name&labels=&category=
**user:** all
**expected behaviour:** If I use the screenreader and look at the proposals in list view, i want that the screenreader also reads me the "Merkmale"-tiles and "Kategorie"-tiles inside the proposal tiles 
**behaviour:** JAWS and NVDA doesnt read the "Merkmale"-tiles and "Kategorie"-tiles that are inside the proposal tiles 
**important screensize:**non
**device & browser:** JAWS (Chrome,Edge,Firefox) and NVDA (Chrome)
**Comment/Question:**

**Comment by phillimorland** [comment]
@SandisStudi is this from tabbing or reading through, when tabbing you only wish to read the interactive elements so reading of the labels would not be expected, only the link

**Comment by fuzzylogic2000** [comment]
@SandisStudi Did you get that when teabbing or reading through? (see comment above)

**Comment by SandisStudi** [comment]
it was when tabbing through, havent tried with reading through yet

**Comment by phillimorland** [comment]
@SandisStudi so there is a 'browse' and a 'focus' mode for screen readers. It is important to check if all interactive elements are accessible via tabbing as there are many keyboard users, however when checking SR navigation, it is mostly done via the a11y tree and walking through content. I only recently found this out but I found this article useful: https://www.accessibility-developer-guide.com/knowledge/screen-readers/desktop/reading-websites/
Also @CarolingerSeilchenspringer might be relevant for you too.
